go.mod

```diff
@@ -24,34 +25,36 @@ require (
 	github.com/mattn/go-colorable v0.1.13
 	github.com/mattn/go-isatty v0.0.16
 	github.com/mccutchen/go-httpbin v1.1.2-0.20190116014521-c5cb2f4802fa
-	github.com/mstoykov/atlas v0.0.0-20220808085829-90340e9998bd
+	github.com/mstoykov/atlas v0.0.0-20220811071828-388f114305dd
 	github.com/mstoykov/envconfig v1.4.1-0.20220114105314-765c6d8c76f1
 	github.com/nu7hatch/gouuid v0.0.0-20131221200532-179d4d0c4d8d
 	github.com/oxtoacart/bpool v0.0.0-20190530202638-03653db5a59c
 	github.com/pmezard/go-difflib v1.0.0
 	github.com/serenize/snaker v0.0.0-20201027110005-a7ad2135616e
 	github.com/sirupsen/logrus v1.9.0
-	github.com/spf13/afero v1.1.2
+	github.com/spf13/afero v1.9.2
```

Comment:
I just used `go get github.com/grafana/xk6-browser@v0.8.0`, and these automatically updated. Is there a way to avoid that? I thought the reason these updated was because of some other dependency requiring that version.

I'll try to undo some of the dep updates.

Reply:
That dependency is likely xk6-browser.  If you can pin the versions for afero and the `x/*` dependencies to the old ones it will likely be better. I would even recommend doing it for atlas as I don't think it is good idea to bump dependencies as close to the end of the cycle as now. 

And AFAIK none of the updates are necessary or urgent for neither project. If that turns out to not be the case we can think about it then IMO

Reply:
:+1: I'd also request what @mstoykov suggests. If these dependency updates are indeed necessary, I'd prefer to handle them in a separate k6 PR, so the diff here is smaller and focused only on xk6-browser.

Reply:
I understand why this is being asked. Is there a way to avoid doing it for the `x/*`? Is there a good reason to pin them to an older version?

Reply:
I think we just forgot to update them on the k6 side at the start of the cycle :disappointed: Personally, I'd only slightly prefer to keep the old versions, but I'd also be fine with making a separate PR to update them in k6 as well. 

We still have nearly 3 weeks before the v0.43.0 release, so it should be relatively safe.

Reply:
As a team we've agreed to revert the update of the dependencies. I've created the PR [#743](https://github.com/grafana/xk6-browser/pull/743). Would be good if you could also take a quick look to make sure I've not missed anything.

Reply:
it's very difficult to check that in the PR, the best way would be to update this PR with xk6-browser from that branch and see how the diff changes, i.e. something like 
```
go get github.com/grafana/xk6-browser@revert/dependency-versions
go mod tidy
go mod vendor
```

Reply:
Already did that and it works
Notes;
1. you will need to checkout `go.mod` from k6's master. As otherwise you will just be changing on top of the already updated dependancies
2. `go get` does not like "commitish" stuff that have `/` as it has separate meaning for modules ;) . SO you need to use the commit

Reply:
Resolved in 317633ab97cd775d501b6a8f3a13b20b3ef60575